website/src/components/FAQ/readme.mdx

```diff
@@ -229,4 +198,7 @@ remains inaccessible from the internet, with no visible entry points.
 
 Firezone helps improve organizations’ cybersecurity posture by offering a modern
 approach to securing access to sensitive Resources in their private network.
```

Comment:
nit: I get "Gateway" and "Client" if it's short for "Firezone Gateway" and "Firezone Client" but this isn't really a proper noun, right?

```suggestion
approach to securing access to sensitive resources in their private network.
```

Reply:
Yeah I was trying to determine how to think about that. In Firezone context it's a proper noun, but since here we preface it with "sensitive" then I think that makes it a generic noun.